WPMedia/fusion-news-theme-blocks
# TMEDIA-721: Alert Bar Block

## Description

Adding the alert-bar-block to the v2 release.

Notable changes:
- Removed unused `has-alert` class support
- Removed hidden flag.  Race condition, only there to delay presentation a half second.
- Presentational component not only takes values instead of the `content` object
- Alert bar now uses the "Danger" theme setting colors instead of the hardcoded red background.

## Jira Ticket

- [TMEDIA-721](https://arcpublishing.atlassian.net/browse/TMEDIA-721)

## Acceptance Criteria

Alert Bar block is migrated to the new architecture.
  * Includes Custom fields of:
    * ‘Arial-label' - open field with default of “Breaking News Alert” and existing help text

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout tmedia-721`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/alert-bar-block,@wpmedia/alert-bar-content-source-block`
3. Open local pagebuilder (http://localhost/pagebuilder/pages) and add alert-bar component in the Daily Intelligencer and notice the alert bar show up. 
4. Run `npm run storybook` and open http://localhost:60001/?path=/story/blocks-alert-bar--headline-and-link
5. Verify News now matches the standard "danger" color of the theme.

## Dependencies or Side Effects

https://github.com/WPMedia/arc-themes-feature-pack/pull/270
https://github.com/WPMedia/arc-themes-feature-pack/pull/273
https://github.com/WPMedia/arc-themes-manifests/pull/66

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
